BE/src/main/java/com/codesquad/issuetracker/auth/response/AccessTokenResponse.java

```diff
@@ -0,0 +1,51 @@
+package com.codesquad.issuetracker.auth.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSetter;
+
+public class AccessTokenResponse {
+
+    @JsonProperty("access_token")
+    private String accessToken;
+
+    private String scope;
+
+    @JsonProperty("token_type")
+    private String tokenType;
+
+    public AccessTokenResponse() {
+
+    }
```

Comment:
```suggestion
    public AccessTokenResponse() {}
```

사용하지 않는 빈 라인은 제거해주어도 좋고, 블럭 내부가 비어있는 경우에는 한 줄로 써줘도 무방합니다.